dotnet/src/SemanticKernel.Abstractions/Data/TextSearch/ITextSearch.cs

```diff
@@ -18,70 +17,72 @@ public interface ITextSearch<TRecord>
     /// </summary>
     /// <param name="query">What to search for.</param>
     /// <param name="searchOptions">Options used when executing a text search.</param>
-    /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
-    Task<KernelSearchResults<string>> SearchAsync(
+    /// <param name="cancellationToken">The <see cref="System.Threading.CancellationToken"/> to monitor for cancellation requests. The default is <see cref="System.Threading.CancellationToken.None"/>.</param>
+    System.Threading.Tasks.Task<KernelSearchResults<string>> SearchAsync(
```

Comment:
Why is this change being done? It doesn't seem related to this PR - please revert. If something here is needed, add using System.Threading.Tasks at the top of the file instead of fully qualifying each time.